### PR TITLE
Matt road test microdc init

### PR DESCRIPTION
**Pull request by fatal-exception**

### CODE REVIEW

**Comment by fatal-exception** [comment]
tested using pip built off branch

File: README.md [hunk 1]
```diff
@@ -59,6 +59,8 @@ microdc --workdir ~/.microdc --config config.yml --account nonprod --stack servi
 ```
 
 5. Validate our setup.
+ Please you may need to update the NS records if using a delegated Route 53 DNS subdomain. The referencing DNS
+ domain should be updated to use the NS servers of the new, terraform-created referenced subdomain.
```
Comment (cherwin) [inline README.md]: Using the imperative mood will improve the readability of this sentence

File: microdc/output_commands.py [hunk 2]
```diff
@@ -219,6 +219,10 @@ def get_k8s_cluster_elb(environment):
                      '        --query \\',
                      '\'LoadBalancerDescriptions[?starts_with(LoadBalancerName, `api-{environment}-`) == `true`].[DNSName]\' --output text)'])  # noqa: E501
           .format(environment=environment))
+    print("\n".join(
+        ["if [[ -z $K8S_API_ELB ]] ; then echo 'ERROR: No Load-Balancer found. Did Kops complete correctly?' 2>&1",
+         "exit 1 ; fi"
+         ]))
```
Comment (cherwin) [inline microdc/output_commands.py]: You might want to consider using a here-doc

**Review by cherwin — COMMENTED** [review]
Overall LGTM

**Comment by fatal-exception** [comment]
PR updated, can you approve again @cherwin

**Review by cherwin — APPROVED** [review]

**Pull request merged**